src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -155,7 +152,27 @@ const StandardView = ({visToolbar, title, searchDesc, standard, imagePlot, xyPlo
             { visToolbar &&
                 <div style={{display: 'inline-flex', justifyContent: 'space-between', alignItems: 'center'}}>
                     <div>{visToolbar}</div>
-                    <div><DownloadButton/></div>
+                    <div>
+                        <DownloadButton>
+                            <DownloadOptionPanel
+                                cutoutSize = '200'
+                                dlParams = {{
+                                    MaxBundleSize: 200*1024*1024,    // set it to 200mb to make it easier to test multi-parts download.  each wise image is ~64mb
+                                    FilePrefix: 'WISE_Files',
+                                    BaseFileName: 'WISE_Files',
+                                    DataSource: 'WISE images',
+                                    FileGroupProcessor: 'LightCurveFileGroupsProcessor'
+                                }}>
+                                <ValidationField
+                                        initialState= {{
+                                               value: 'A sample download',
+                                               label : 'Title for this download:'
```

Comment:
Is it possible to make the default title more descriptive, for example, by including the type (cutout or image) and the number of files to be downloaded?